tools/@aws-cdk/generate-examples/test/testutil.ts

```diff
@@ -46,20 +46,22 @@ export class AssemblyFixture {
 }
 
 export const DUMMY_ASSEMBLY_TARGETS = {
-  dotnet: {
-    namespace: 'Example.Test.Demo',
-    packageId: 'Example.Test.Demo',
-  },
-  go: { moduleName: 'example.test/demo' },
-  java: {
-    maven: {
-      groupId: 'example.test',
-      artifactId: 'demo',
+  targets: {
```

Comment:
This [commit](https://github.com/aws/jsii/pull/3233/files#diff-89d69745cce42cdf5abd3a61d2612a4015406c3bd29a5f1c2460b0e1875bc165) in jsii made some loosely defined schemas be type-safe. Which caught the fact that this wasn't typed properly